app/assets/javascripts/items.js.coffee

```diff
@@ -4,3 +4,15 @@ jQuery ->
     regexp = new RegExp($(this).data('id'), 'g')
     $(this).before($(this).data('fields').replace(regexp, time))
     event.preventDefault()
+
+jQuery ->
+  $('#items').dataTable
+    columnDefs: [
+                  ## https://datatables.net/extensions/responsive/priority
```

Comment:
I really appreciate this comment, makes review easier. And "future us" will probably use it too.